### PR TITLE
Issue #244: Deposit TVL Flow

### LINKED ISSUE #244: Deposit flow
Project #234 

Create the approve + deposit flow. When possible try to use existing modal code.

image

### PR DESCRIPTION
**Pull request by kai-thompson**
Closes #244 

## Description
- Add styles to slider component that matches that of the screenshot in related issue
- Add DepositFunds component and view
    - Dates are hardcoded
    - onDeposit method isn't implemented
    - onInfoClick method isn't implemented

### TVL Deposit
_Note: Right label in TokenInput now is the deposit amount in USD_

https://github.com/open-dollar/od-app/assets/90171376/b62b84c0-cb57-47a4-b496-347cd1bf131d

### Slider(Chrome)
https://github.com/open-dollar/od-app/assets/90171376/a856c025-5b15-4c1d-a183-986d5dbbf2b3
### Slider(Safari)
https://github.com/open-dollar/od-app/assets/90171376/f27e57ff-b791-4c35-ae0b-df028fda5122
### Slider(Firefox)
https://github.com/open-dollar/od-app/assets/90171376/8b01b99a-3439-4fb7-befb-19ab1ecb0cd1

### CODE REVIEW

**Comment by kai-thompson** [comment]
Questions:
- Do we want to just leverage the modal component for stylistic purposes but have it act as a normal component OR do we want it to actually be a modal?

**Comment by kai-thompson** [comment]
image
What should the text in the blue info icon be?

**Comment by kai-thompson** [comment]
Should I use liquidation data to get the USD value of collateral balance?

**Comment by kai-thompson** [comment]
Is there a way to get the dates from a contract? Or are they just always hard-coded values?

**Review by jahabeebs — APPROVED** [review]
Looks great, wait for Pat's review though

File: src/App.tsx [hunk 1]
```diff
@@ -78,6 +79,12 @@ const App = () => {
                                                 <Route exact component={VaultDetails} path={'/vaults/:id'} />
                                                 <Route exact strict component={Safes} path={'/vaults'} />
                                                 <Route exact strict component={Safes} path={'/:address'} />
+                                                <Route
+                                                    exact
+                                                    strict
+                                                    component={DepositFunds}
+                                                    path={'/tvl/:token/deposit'}
```
Comment (pi0neerpat) [inline src/App.tsx:86]: route should be `/deposit/:token/deposit`

**Review by pi0neerpat — APPROVED** [review]

**Pull request merged**